### DISCUSSION TITLE
Designing AF's Contributor T-Shirts

### DISCUSSION THREAD

**Discussion by annieappflowy** [AppFlowy-IO/AppFlowy#214] [General]
I'm getting contributor t-shirts for the community. Here is the original design by our design agency. I'm open to suggestions on colors. Please feel free to voice your opinion! Thanks!
![image](https://user-images.githubusercontent.com/12026239/148350379-678fd9db-540c-4c6a-9246-9ea9b001495e.png)

**Comment by MikeWallaceDev** [comment]
I'm not a fan of this one. The "design team" just slapped the icon in the middle of a t-shirt.

I'm going to submit a few ideas.  They are not meant to be final choices, but they are a good starting point.  Maybe they will inspire someone with actual talent can make something better.

**Comment by MikeWallaceDev** [comment]
![image](https://user-images.githubusercontent.com/5375835/148418211-59a8397d-66ae-4915-be6e-48e9ad6b08ff.png)

**Comment by MikeWallaceDev** [comment]
![image](https://user-images.githubusercontent.com/5375835/148418407-2298a24d-9698-49af-a10b-95b552383eb1.png)

**Comment by MikeWallaceDev** [comment]
![image](https://user-images.githubusercontent.com/5375835/148418480-96289da2-4d62-4fd6-a633-022d688f9096.png)

**Comment by MikeWallaceDev** [comment]
![image](https://user-images.githubusercontent.com/5375835/148418545-0f094640-8282-4104-84e3-ba589eb86ed6.png)

**Reply by lawweiliang** [reply]
This one look goods. 😃

**Comment by MikeWallaceDev** [comment]
![image](https://user-images.githubusercontent.com/5375835/148418618-fd9fb470-e782-41da-98f2-9ff34d62d818.png)
